### PR TITLE
Update new_library.md

### PR DESCRIPTION
**Pull request by dvyukov**
scripts/helper.py should be run with sudo, otherwise it fails with:
```
Running: docker build -t ossfuzz/...
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
docker build failed.
```

### CODE REVIEW

File: docs/new_library.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ version using docker commands directly is documented [here](building_running_fuz
 
 ```bash
 $ cd /path/to/oss-fuzz
-$ python scripts/helper.py build_image $LIB_NAME
-$ python scripts/helper.py build_fuzzers $LIB_NAME
+$ sudo python scripts/helper.py build_image $LIB_NAME
```
Comment (mikea) [inline docs/new_library.md]: sudo? Why did you need it?

Does docker run without sudo for you? (docker ps)

Comment (mikea) [inline docs/new_library.md]: you need to reboot computer or run `newgrp` in your session. @eugenis can you verify?

Comment (eugenis) [inline docs/new_library.md]: yes
don;t know about newgrp but reboot helps

Comment (dvyukov) [inline docs/new_library.md]: I did not read/follow past "Optional configurations". Mailed https://github.com/google/oss-fuzz/pull/28 to clarify instructions.
